homeassistant/components/emoncms/sensor.py

```diff
@@ -210,7 +209,6 @@ def _update_attributes(self, elem: dict[str, Any]) -> None:
         elif elem["value"] is not None:
             self._attr_native_value = round(float(elem["value"]), DECIMALS)
 
-    @Throttle(MIN_TIME_BETWEEN_UPDATES)
```

Comment:
MIN_TIME_BETWEEN_UPDATES can be removed now

Reply:
Yes, just realized it